src/Symfony/Bundle/FrameworkBundle/Logger/Logger.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Logger;
+
+use Psr\Log\AbstractLogger;
+use Psr\Log\InvalidArgumentException;
+use Psr\Log\LogLevel;
+
+/**
+ * Minimalist PSR-3 logger designed to write in stdout, stderr or any other stream.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ *
+ * @see http://www.php-fig.org/psr/psr-3/
+ */
+final class Logger extends AbstractLogger
+{
+    /**
+     * @var array
+     */
+    private static $defaultLevelToStream = array(
+        LogLevel::EMERGENCY => 'php://stderr',
+        LogLevel::ALERT => 'php://stderr',
+        LogLevel::CRITICAL => 'php://stderr',
+        LogLevel::ERROR => 'php://stderr',
+        LogLevel::WARNING => 'php://stdout',
+        LogLevel::NOTICE => 'php://stdout',
+        LogLevel::INFO => 'php://stdout',
```

Comment:
I haven't had a look carefully, but I'm not sure we need to output info/notice logs. Intuitively, I would have skipped them.